events.html

```diff
@@ -16,3 +16,55 @@
         </div>
     </div>
 </div>
+<div class="container list-events">
+    <section class="row col-12">
+        <div class="row-header col-12">
+            <h2>Events</h2>
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a class="nav-link active">Upcoming Events</a>
+                </li>
+                <li class="nav-item"> 
+                    <a class="nav-link disabled">Past Events</a>
+                </li>
+            </ul>
+        </div>
+        <div class="row-content col-12">
+            <div class="card-deck">
+                {% assign count = 0 %}
+                {% for events in site.data.events %}
+                    {% assign count = count | plus: 1 %}
+                    {% if count <= 2 %}
```

Comment:
this if can be removed, becase on this page we want to show all events. You can keep here the for only, because everything else is not necessary.

Reply:
Again, I think this should be done in a different pr 🙈 because the other logic could be already merged, it is very good!

Reply:
but, you can merge like this and I can make the changes for you if you want.

Reply:
Ah yes, thanks for the suggestion, I've corrected it :)

And yes, sorry, will remember to create separate PRs next time.